test/erc1271-order.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { expect } from "chai";
+import { parseEther } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { ItemType, MAX_INT } from "../src/constants";
+import { CreateOrderAction } from "../src/types";
+import { generateRandomSalt } from "../src/utils/order";
+import { describeWithFixture } from "./utils/setup";
+
+const OPENSEA_DOMAIN = "opensea.io";
+const OPENSEA_TAG = "360c6ebe";
+
+describeWithFixture(
```

Comment:
instead of a new test file, could you just add the test to the existing create-order.spec.ts?